cg/meta/workflow/balsamic.py

```diff
@@ -238,22 +237,29 @@ def get_verified_pon(self, panel_bed: str, pon_cnn: str) -> str | None:
                 raise BalsamicStartError(
                     f"The specified PON reference file {latest_pon} does not match the panel bed {panel_bed}"
                 )
+            latest_pon = self.get_verified_sex_pon(sample_sex=sex, pon_cnn=latest_pon)
+
```

Comment:
Wouldn't this logic be simpler?

```
        if pon_cnn:
            panel_name: str = Path(panel_bed).stem
            pon_name: str = Path(pon_cnn).stem
            if panel_name not in pon_name:
                raise BalsamicStartError(
                    f"The PON reference file {pon_cnn} does not match the panel bed {panel_bed}"
                )
            if sex not in pon_name and (sex.MALE in pon_name or sex.FEMALE in pon_name):
                raise BalsamicStartError(
                    f"The PON reference file {pon_cnn} does not match the sample sex {sex}"
                )
            latest_pon: str = pon_cnn
        else:
            latest_pon: str | None = self.get_latest_pon_file(panel_bed=panel_bed, sex=sex)
            if latest_pon:
                LOG.info(
                    f"The following PON reference file will be used for the analysis: {latest_pon}"
                )
        return latest_pon
```

```suggestion
```

Reply:
Yes! Much better. I'll change the logic : )